doc/format-doc.py

```diff
@@ -199,21 +198,15 @@ def run(self, clsname, path=[]):
 def main():
     parser = argparse.ArgumentParser(description='Generate object documentation')
     parser.add_argument('jsondoc', help='the hlwm-doc.json file')
+    parser.add_argument('--objects-intro',
```

Comment:
Is this parameter used anywhere? I cannot find it

Reply:
That's a leftover from a previous approach. I've removed it.

Reply:
Thanks!